doc/topics/matchers/index.rst

```diff
@@ -0,0 +1,81 @@
+.. _matchers:
+
+========
+Matchers
+========
+
+.. versionadded:: Flourine
+
+Matchers are modules that provide Salt's targeting abilities.  As of the
+Flourine release, matchers can be dynamically loaded.  Currently new matchers
+cannot be created, but existing matchers may have their functionality altered or
```

Comment:
Is this just because the CLI would need to be modified as well? I feel like this needs more explanation.